connection.go

```diff
@@ -232,8 +235,13 @@ func (sc *snowflakeConn) ExecContext(ctx context.Context, query string, args []d
 	if sc.rest == nil {
 		return nil, driver.ErrBadConn
 	}
-	// TODO: handle noResult and isInternal
-	data, err := sc.exec(ctx, query, false, false, args)
+	noResult, err := isAsyncMode(ctx)
+	if err != nil {
+		return nil, err
+	}
+	// TODO handle isInternal
```

Comment:
Aren't we handling isInternal already? Above?

Reply:
The feature isn't in place yet -- we just pass in the isInternal value but it's false by default currently.